src/components/Shared/withSubscription.tsx

```diff
@@ -68,7 +68,8 @@ interface IWithSubscriptionOptions<Props extends ISubscriptionProps<ObservableTy
  * required by the wrappedComponent and passes along the data observed as it comes in.
  * @oarams options See `IWithSubscriptionOptions`
  */
-const withSubscription = <Props extends ISubscriptionProps<ObservableType>, ObservableType extends Record<string, any>>(options: IWithSubscriptionOptions<Props, ObservableType>) => {
+
+const withSubscription = <Props extends ISubscriptionProps<ObservableType>, ObservableType extends Record<string, any>>(options: IWithSubscriptionOptions<Props, ObservableType>): any => {
```

Comment:
This returns:

`new (...args: any) => React.Component<OnlyWrappedComponentProps<Props>, IObservableState<ObservableType>>`

Reply:
When trying this return value I get errors in `CreateProposalPage.tsx`